contrib/test/test-vectors-fixtures/cpi-fixtures/cpi.list

```diff
@@ -1,2 +1,3 @@
 dump/test-vectors/cpi/fixtures/callee_not_executable_log.fix
 dump/test-vectors/cpi/fixtures/cpi_prepare_missing_acct.fix
+dump/test-vectors/cpi/fixtures/18387a7e499df53e20db216f4520bca9ce0727e7_1582213.fix
```

Comment:
In future the names of these fixtures should describe what exactly they test for (see above)